workspaces/global-header/plugins/global-header/src/components/MenuItemLink/MenuItemLinkContent.tsx

```diff
@@ -13,64 +13,79 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 import React from 'react';
 import Typography from '@mui/material/Typography';
 import Box from '@mui/material/Box';
 import { HeaderIcon } from '../HeaderIcon/HeaderIcon';
 import { useTheme } from '@mui/material/styles';
+import LaunchIcon from '@mui/icons-material/Launch';
 
 interface MenuItemLinkContentProps {
   icon?: string;
   label?: string;
   subLabel?: string;
+  isExternalLink?: boolean;
 }
 
 export const MenuItemLinkContent: React.FC<MenuItemLinkContentProps> = ({
   icon,
   label,
   subLabel,
+  isExternalLink = false,
 }) => {
   const theme = useTheme();
   return (
     <Box
       sx={{
         display: 'flex',
         alignItems: 'center',
+        justifyContent: 'space-between',
         margin: '8px 0',
         color: 'inherit',
+        width: '100%',
       }}
     >
-      {icon && (
-        <HeaderIcon
-          icon={icon}
-          size="small"
-          layout={
-            label
-              ? {
-                  marginRight: '0.5rem',
-                  flexShrink: 0,
-                  color:
-                    theme.palette.mode === 'dark'
-                      ? theme.palette.text.primary
-                      : theme.palette.text.disabled,
-                }
-              : {}
-          }
-        />
-      )}
-      <Box>
-        {label && (
-          <Typography variant="body2" color={theme.palette.text.primary}>
-            {label}
-          </Typography>
-        )}
-        {subLabel && (
-          <Typography variant="caption" color="text.secondary">
-            {subLabel}
-          </Typography>
+      <Box display="flex" alignItems="center">
+        {icon && (
+          <HeaderIcon
+            icon={icon}
+            size="small"
+            layout={
+              label
+                ? {
+                    marginRight: '0.5rem',
+                    flexShrink: 0,
+                    color:
+                      theme.palette.mode === 'dark'
+                        ? theme.palette.text.primary
+                        : theme.palette.text.disabled,
+                  }
+                : {}
+            }
+          />
         )}
+        <Box>
+          {label && (
+            <Typography variant="body2" color={theme.palette.text.primary}>
+              {label}
+            </Typography>
+          )}
+          {subLabel && (
+            <Typography variant="caption" color="text.secondary">
+              {subLabel}
+            </Typography>
+          )}
+        </Box>
       </Box>
+      {isExternalLink && (
+        <LaunchIcon
+          sx={{
+            fontSize: 16,
+            color: theme.palette.text.secondary,
+            marginLeft: '0.5rem',
```

Comment:
We can use sx attributes here to use the theme, but it's up to you, both works.

```suggestion
            ml: 1,
```

Reply:
Done.